build.gradle

```diff
@@ -183,6 +184,16 @@ jar {
 	}
 }
 
+task generateTestMappings(type: JavaExec, dependsOn: testClasses) {
+	classpath sourceSets.test.runtimeClasspath
+	mainClass = 'net.fabricmc.mappingio.TestFileUpdater'
+}
+
+task updateTestMappings(type: Copy, dependsOn: generateTestMappings) {
+	from 'build/resources/test/'
+	into 'src/test/resources/'
+}
```

Comment:
`tasks.register` is preferred with modern Gradle.

Reply:
Done